branding.html

```diff
@@ -156,6 +156,13 @@ <h2>Lucy, Gleam's starfish mascot</h2>
       alt="Lucy but the colours of the men-loving-men pride flag instead of pink"
     />
   </li>
+  <li>
+    <img
+      src="/images/lucy/lucyace.svg"
+      title="Lucy ace"
```

Comment:
Don't want to credit yourself? :)

Reply:
nah, don't really mind. got a few Lucy's added by this point and it'd feel weird to start now hahah